rust/benchmarks/src/bin/tpch.rs

```diff
@@ -59,6 +61,10 @@ struct TpchOpt {
     /// File format: `csv` or `parquet`
     #[structopt(short = "f", long = "format", default_value = "csv")]
     file_format: String,
+
+    /// Load the data into a MemTable before executing the query
+    #[structopt(short = "l", long = "load")]
```

Comment:
There is probably a better/clearer name for this parameter

Reply:
Perhaps `--mem-table` ?